.github/CONTRIBUTING.md

```diff
@@ -6,9 +6,9 @@ consistent workflow.
 ## Table of Contents
 
 1. [Branch Naming Convention](#branch-naming-convention)
-2. [Commit Message Guidelines](#commit-message-guidelines)
+2. [Commit Message Guidelines](#pr-types)
```

Comment:
Thanks for pointing out the issue. Could you help remove this line? We will update the Commit Message Guidelines later. 

The #versioning tag is fine.